codecov/urls.py

```diff
@@ -24,6 +24,16 @@
     path(f"{settings.DJANGO_ADMIN_URL}/", admin.site.urls),
     path("staticanalysis/", include("staticanalysis.urls")),
     path("labels/", include("labelanalysis.urls")),
+    path(
+        "admin-repository-autocomplete/",
+        views.RepositoryAutoCompleteSearch.as_view(),
+        name="admin-repository-autocomplete",
+    ),
+    path(
```

Comment:
these endpoints will be exposed on the API but I think its fine cause of [this](https://github.com/codecov/codecov-api/blob/ca8de7b5d4334c1dd0122efab355d0ccfb91b328/codecov/views.py#L37)

Reply:
thanks for calling this out!